enhancements/sig-policy/98-long-term-compliance-history/README.md

```diff
@@ -246,6 +246,8 @@ The following optional query parameters would be accepted on this HTTP endpoint:
 - after - an ISO-8601 timestamp to indicate only compliance events after this time should be shown.
 - page - the page number in the query. This defaults to 1.
 - per_page - the number of compliance events returned per page. This defaults to 20 and maxes out at 100.
+- sort - the field to sort by. This defaults to `timestamps`.
```

Comment:
Should it mention which fields will be supported here?

Reply:
The code I have locally just allows sorting on any field that is a database column.

Reply:
Is sorting done with only one field?

Reply:
I originally had it so that you'd specific `?sort=policy.name&sort=policy.namespace` but I think I'll just use commas as a separator to match what I'm planning for filters. I'll update this.